node/tests/TestUtilities.ts

```diff
@@ -439,8 +441,19 @@ export async function transactionTest(
     args.push("OK");
     baseTransaction.lrange(key5, 0, -1);
     args.push([field + "3", field + "2"]);
+
+    if (gte("6.2.0", version)) {
+        baseTransaction.lmove(
+            key5,
+            key19,
+            ListDirection.LEFT,
+            ListDirection.LEFT,
+        );
+        args.push(field + "3").toString();
+    }
+
     baseTransaction.lpopCount(key5, 2);
-    args.push([field + "3", field + "2"]);
+    args.push([field + "2"]);
```

Comment:
we should test this change with a version of redis less than 6.2...

Reply:
how do we do this?
through local testing or add a part in the test utitlities?